src/dune/__main__.py

```diff
@@ -93,6 +94,18 @@ def load_all_modules_from_dir(plugin_dir):
             WAS_REMAINDER_ARGS_USED = False
 
             if args.start is not None:
+                if args.rmdirtydb is True:
+                    node_state_dir = '/app/nodes/' + args.start[0] +'/state/'
+                    if dune_sys.docker.dir_exists(node_state_dir) and not dune_sys.is_named_node_running(args.start[0]):
+                        print("Removing container directory " + node_state_dir
+                            + " to clean a database dirty flag of node [" + args.start[0] + "]...\n")
+                        dune_sys.docker.execute_cmd(['rm', '-rf', node_state_dir])
```

Comment:
What happens if the call fails?

Reply:
The consensus on the leap team is nodeos w/ --replay-blockchain is preferable to rm -r. Let's switch, please.

Reply:
OK, replay-blockchain option is used